EventsByTag.md

```diff
@@ -85,6 +85,14 @@ In this case the eventsByTag query looks for the new-persistence-id-scan-timeout
 is found by an offset query. If this is an issue it can be set to 0s. If events are found out of order due to this
 the stage will fail.  
```

Comment:
Can we clarify when an event can be missed? After the `new-persistence-id-scan-timeout` it will use the found seqNr as the first event, but when is there a risk that it will miss an event anyway when starting from a stored offset?